content/posts/2023/reana-0.9.1.md

```diff
@@ -0,0 +1,109 @@
+
+## What's new for the administrators?
+
+### Keycloak
+
+REANA provides users with two ways to log in, either using username/password or via an external SSO provider.
+Up until now support for external SSO systems was limited to CERN SSO, but REANA 0.9.1 removes this limitation by being able to integrate with any custom Keycloak instance.
+
+If you are interested in using Keycloak together with REANA, please take a look to the related [Keycloak Single Sign-On configuration](https://docs.reana.io/administration/configuration/configuring-access/#keycloak-single-sign-on-configuration) documentation page.
+
+### Launcher examples
+
+REANA 0.9.0 introduced the possibility to launch workflows from external sources like GitHub, GitLab or Zenodo.
+It is now possible to customise the demo examples that are showcased in the Launch-on-REANA welcome page via the new `components.reana_ui.launcher_examples` Helm value.
+In this way you can expose your own demos to your users, instead of the default ones provided by REANA.
+
+![Launcher examples](/images/reana-0.9.1-launcher-examples.png)
+
+<!-- TODO: add link to docs/blog post here -->
+
+### Automatic closure of inactive sessions
+
+It might happen that users open some interactive sessions and forget to close them when they are not needed anymore.
+To keep resource utilisation under control, REANA 0.9.1 allows to set a maximum time period of inactivity after which inactive sessions will be closed.
+The inactivity period can be customised with the `interactive_sessions.maximum_inactivity_period` Helm value.
+
+To learn more about this feature, please have a look at the related [Configuring interactive sessions](https://docs.reana.io/administration/configuration/configuring-interactive-sessions/#auto-closure-of-inactive-sessions) documentation page.
+
+### Performance and stability improvements
+
+There are many improvements in performance and stability that have landed in this new REANA release:
+
+- The periodic quota updater for disk and CPU quotas is now up to 70% faster.
+- The memory usage of uWSGI and RabbitMQ has been reduced in in environments with a very high number of allowed open files.
+- Static assets of the web interface, such as CSS files and bundled JS sources, are now also served gzip-compressed to lower bandwidth consumption.
+- Monitoring the status of jobs has been improved to avoid some situations in which workflows were not correctly cleaned up in the Kubernetes cluster.
+
+### ... and much more!
+
+Additional features and improvements are present in REANA 0.9.1.
+It is now possible to customise the file size limit for files that can be previewed form the web UI, so that it can be increased or decreased as necessary, for example to allow previewing large ROOT files.
+There are also improvements to how emails are sent, as it is now possible to chose whether to enable SSL or STARTTLS.
```

Comment:
The email sending feature was also improved and you can now customise whether to enable SSL or STARTTLS.